app/enums.py

```diff
@@ -1,4 +1,4 @@
-from strenum import StrEnum  # type: ignore [import-not-found]
+from enum import StrEnum  # type: ignore [import-not-found]
```

Comment:
you can remove the type: ignore comment.

Also, the comment below this can be removed, as it simply is describing what you did here.